clang/lib/Sema/SemaChecking.cpp

```diff
@@ -3005,6 +3005,15 @@ enum ArmStreamingType {
   ArmStreamingOrSVE2p1
 };
 
+enum ArmSMEState : unsigned {
+  ArmNoState = 0,
+
+  ArmInZA = 0b01,
+  ArmOutZA = 0b10,
+  ArmInOutZA = 0b11,
+  ArmZAMask = 0b11,
```

Comment:
```suggestion
  ArmZAMask = ArmInOutZA,
```

Reply:
I'm not really convinced that's better because they're two different things. One is an enum value for the state of ZA, and the other is a bitmask. If we'd want to change the encoding of the ZA attributes, we don't necessarily have to update the mask.